modules/document_repository/php/document_repository.class.inc

```diff
@@ -79,9 +79,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Date_uploaded',
                          );
 
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = !empty($_GET['openAccordion']);
         $this->tpl_data['openAccordion'] = $openAccordion;
-        $filtered = $_GET['filtered'];
+        $filtered = !empty($_GET['filtered']);
```

Comment:
@intralizee @zaliqarosli @driusan Did more testing on this and this will always cause filtering by default (not desired). The smarty template only checks whether `tpl_data['filtered']` is set; it doesn't care whether it's true or false.

Two options here:
1. Revert to my change in #3603 where we set this variable to null if it is not present as a parameter
2. Change the template to pass along the boolean value of `$filtered` instead of just checking if it's set.

I prefer option 2.

Reply:
@johnsaigle Okay so there is no reason to check for isset since $filtered should always be either true or false. I modified the smarty template to remove isset. Thoughts?

Reply:
Looks good